### PR TITLE
Edits off auto poll

### PR DESCRIPTION
**Pull request by DrewAns**
No Ticket.

Fix polling hanging spinner

### CODE REVIEW

File: ui/src/app/services/api/embassy-api.service.ts [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 export abstract class ApiService implements Source<DataModel>, Http<DataModel> {
   protected readonly sync$ = new Subject<Update<DataModel>>()
 
   /** PatchDb Source interface. Post/Patch requests provide a source of patches to the db. */
   // sequenceStream '_' is not used by the live api, but is overridden by the mock
-  watch$ (_?: Store<DataModel>): Observable<Update<DataModel>> {
-    return this.sync$.asObservable()
+  watch$ (_?: Store<DataModel>): Observable<RPCResponse<Update<DataModel>>> {
+    return this.sync$.asObservable().pipe(map( result => ({ result,
+      jsonrpc: '2.0'})))
```
Comment (Blu-J) [inline ui/src/app/services/api/embassy-api.service.ts:15]: I did this to convert the type back into a RPCResponse.

File: ui/src/app/services/patch-db/patch-db.service.ts [hunk 2]
```diff
@@ -39,66 +41,95 @@ export class PatchDbService {
...
-          console.log(polling ? 'patchDB: POLL CONNECTED' : 'patchDB: WEBSOCKET CONNECTED')
-          this.patchConnection$.next(PatchConnection.Connected)
-          if (!wsSuccess && !polling) {
-            console.log('patchDB: WEBSOCKET SUCCESS')
-            this.storage.set(this.WS_SUCCESS, 'true')
-            this.wsSuccess$.next(true)
+  async start (): Promise<void> {
+    await this.init()
+
+    console.log('PATCH DB', this.patchDb)
```
Comment (Blu-J) [inline ui/src/app/services/patch-db/patch-db.service.ts:56]: Is this a comment we want?

File: ui/src/app/services/patch-db/patch-db.service.ts [hunk 3]
```diff
@@ -128,22 +150,37 @@ export class PatchDbService {
...
 }
+
+
+function switchMapAfter<A, B> (projection: (a: A) => Observable<B>) {
+  return (observable: Observable<A>) =>
+    new Observable<B>((subscriber) => {
+      let lastLive: null | Subscription = null
+      const subscription = observable.subscribe({
+        next (value) {
+          console.log('switchMapAfter JCWM new events')
```
Comment (Blu-J) [inline ui/src/app/services/patch-db/patch-db.service.ts:171]: Remove comment

File: ui/src/app/services/patch-db/patch-db.service.ts [hunk 4]
```diff
@@ -128,22 +150,37 @@ export class PatchDbService {
...
+          console.error('patchDB: WATCH ERROR', e)
+          return of(e.message)
+        }),
+        finalize(() => console.log('patchDB: UNSUBSCRIBING', argsString)),
+      )
   }
 }
+
+
+function switchMapAfter<A, B> (projection: (a: A) => Observable<B>) {
```
Comment (Blu-J) [inline ui/src/app/services/patch-db/patch-db.service.ts:165]: I think we can remove this function, we are not using it anymore

File: ui/src/app/services/patch-db/patch-db.service.ts [hunk 5]
```diff
@@ -39,66 +41,95 @@ export class PatchDbService {
...
-            this.storage.set(this.WS_SUCCESS, 'true')
-            this.wsSuccess$.next(true)
+  async start (): Promise<void> {
+    await this.init()
+
+    console.log('PATCH DB', this.patchDb)
+
+    this.subs.push(
+
+      // Connection Error
```
Comment (Blu-J) [inline ui/src/app/services/patch-db/patch-db.service.ts:60]: I asked that the signals get split into individual to separate concerns + understand

File: ui/src/app/services/patch-db/patch-db.service.ts (hunk 3 above)
Comment (ProofOfKeags) [reply ui/src/app/services/patch-db/patch-db.service.ts:171]: Maybe it's time for a proper level/sink-based logging library for the FE.

**Pull request merged**